bundles/org.openhab.binding.comfoair/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1465 @@
+			<channel id="bypassState" typeId="bypass_state" />
+			<channel id="ewtState" typeId="ewt_state" />
+			<channel id="heaterState" typeId="heater_state" />
+			<channel id="vControlState" typeId="v_control_state" />
+			<channel id="frostState" typeId="frost_state" />
+			<channel id="cookerhoodState" typeId="cookerhood_state" />
+			<channel id="enthalpyState" typeId="enthalpy_state" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="inputs">
+		<label>Inputs</label>
+		<channels>
+			<channel id="isL1Switch" typeId="is_L1_switch" />
+			<channel id="isL2Switch" typeId="is_L2_switch" />
+			<channel id="isBathroomSwitch" typeId="is_bathroom_switch" />
+			<channel id="isCookerhoodSwitch" typeId="is_cookerhood_switch" />
+			<channel id="isExternalFilter" typeId="is_external_filter" />
+			<channel id="isWTW" typeId="is_wtw" />
+			<channel id="isBathroom2Switch" typeId="is_bathroom2_switch" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="analog">
+		<label>Analog Inputs</label>
+		<channels>
+			<channel id="isAnalog1" typeId="is_analog1" />
+			<channel id="isAnalog2" typeId="is_analog2" />
+			<channel id="isAnalog3" typeId="is_analog3" />
+			<channel id="isAnalog4" typeId="is_analog4" />
+			<channel id="isRF" typeId="is_RF" />
+			<channel id="analog1Mode" typeId="analog1_mode" />
+			<channel id="analog2Mode" typeId="analog2_mode" />
+			<channel id="analog3Mode" typeId="analog3_mode" />
+			<channel id="analog4Mode" typeId="analog4_mode" />
+			<channel id="RFMode" typeId="RF_mode" />
+			<channel id="analog1Negative" typeId="analog1_negative" />
+			<channel id="analog2Negative" typeId="analog2_negative" />
+			<channel id="analog3Negative" typeId="analog3_negative" />
+			<channel id="analog4Negative" typeId="analog4_negative" />
+			<channel id="RFNegative" typeId="RF_negative" />
+			<channel id="analog1Volt" typeId="analog1_volt" />
+			<channel id="analog1Min" typeId="analog1_min" />
+			<channel id="analog1Max" typeId="analog1_max" />
+			<channel id="analog1Value" typeId="analog1_value" />
+			<channel id="analog2Volt" typeId="analog2_volt" />
+			<channel id="analog2Min" typeId="analog2_min" />
+			<channel id="analog2Max" typeId="analog2_max" />
+			<channel id="analog2Value" typeId="analog2_value" />
+			<channel id="analog3Volt" typeId="analog3_volt" />
+			<channel id="analog3Min" typeId="analog3_min" />
+			<channel id="analog3Max" typeId="analog3_max" />
+			<channel id="analog3Value" typeId="analog3_value" />
+			<channel id="analog4Volt" typeId="analog4_volt" />
+			<channel id="analog4Min" typeId="analog4_min" />
+			<channel id="analog4Max" typeId="analog4_max" />
+			<channel id="analog4Value" typeId="analog4_value" />
+			<channel id="RFMin" typeId="RF_min" />
+			<channel id="RFMax" typeId="RF_max" />
+			<channel id="RFValue" typeId="RF_value" />
+			<channel id="analogMode" typeId="analog_mode" />
+		</channels>
+	</channel-group-type>
```

Comment:
Instead of grouping all your analog channels under a single group you could use multiple groups such that you would have numbered analog groups instead of numbered analog channels.
```suggestion
	<channel-group-type id="analogRf">
	    <label>Analog RF Inputs</label>
		<channels>
			<channel id="isRF" typeId="is_RF" />
			<channel id="RFMode" typeId="RF_mode" />
			<channel id="RFNegative" typeId="RF_negative" />
			<channel id="RFMin" typeId="RF_min" />
			<channel id="RFMax" typeId="RF_max" />
			<channel id="RFValue" typeId="RF_value" />
			<channel id="analogMode" typeId="analog_mode" />
		</channels>
	</channel-group-type>
	<channel-group-type id="analog">
		<label>Analog Inputs</label>
		<channels>
			<channel id="isAnalog" typeId="is_analog" />
			<channel id="analogMode" typeId="analog_mode" />
			<channel id="analogNegative" typeId="analog_negative" />
			<channel id="analogVolt" typeId="analog_volt" />
			<channel id="analogMin" typeId="analog_min" />
			<channel id="analogMax" typeId="analog_max" />
			<channel id="analogValue" typeId="analog_value" />
		</channels>
	</channel-group-type>
```

When in your thingtype definition:
```
		<channel-groups>
...
			<channel-group id="analogRf" typeId="analogRf" />
			<channel-group id="analog1" typeId="analog" />
			<channel-group id="analog2" typeId="analog" />
			<channel-group id="analog3" typeId="analog" />
			<channel-group id="analog4" typeId="analog" />
...
		</channel-groups>
```

This should allow you to cleanup a significant number of channel type definitions. WDYT?

Reply:
Good point. Implemented and will push after a short test.

Reply:
done